### DISCUSSION TITLE
Hosting recommendations

### DISCUSSION THREAD

**Discussion by adherb** [nandorojo/solito#92] [Q&A]
Hi, 

I'm super excited to be starting a project with Solito and I just have a couple of questions on hosting recommendations.

1. I can see the starter project can be deployed to Vercel, does this mean that the React Native app can be hosted on vercel and deployed to the App/Play store?
2. If not is there a recommended place to host the monorepo?

The ideal solution for me would be to use Vercel to host the monorepo and Next.js api routes for the backend. But I am open to suggestions, I know AWS amplify would also be an option but It's not compatible with the more recent versions of Next and the builds are painfully slow.

Thanks,
Adam

**Comment by RyelBanfield** [comment]
Hey @adherb, how did the project go? I'm guessing you went with Vercel?

**Reply by adherb** [reply]
Hi @RyelBanfield, I have almost finished it. The frontend Next site I'm using Vercel and for the node backend I'm using AWS, specifically elastic beanstalk. Another option I looked at was Digital Ocean.

The reason I've moved away from using serverless is at scale it gets expensive.
